src/index/import/markdown.ts

```diff
@@ -145,11 +147,24 @@ export function markdownImport(
             list.id,
             list.task
         );
+        let content = lines.slice(item.start, item.end + 1).join("\n");
+        /** strip inline fields maybe */
+
+        let marker = content.split("\n")[0].replace(markerRegex, "").trim().slice(0, 1);
+        item.text = content.replace(contentRegex, "");
+        item.symbol = marker;
```

Comment:
I'd avoid the mutability here - parse this information first and pass it to the list item constructor instead.